src/avram/save_operation.cr

```diff
@@ -379,6 +383,14 @@ abstract class Avram::SaveOperation(T)
     @record.try &.id
   end
 
+  def self.column_names
+    T.column_names
+  end
+
+  def self.database_table_info
+    T.database_table_info
+  end
```

Comment:
Might want to delegate here:
https://crystal-lang.org/api/1.3.0/Object.html#delegate(*methods,toobject)-macro

It seems it might make stack traces more direct

Reply:
Not sure given this is a class method and `delegate` is a macro, but happy to take a look into it more if you think it's worth investigating.

Reply:
I don't think this is a huge deal, it'd just be 1 stack trace jump out of "framework error full traces"